### PR TITLE
Fix 'all' declaration for KeyValueStore

### PR DESCRIPTION
**Pull request by EmiM**

### CODE REVIEW

File: KeyValueStore.d.ts [hunk 1]
```diff
@@ -8,5 +8,7 @@ declare module "orbit-db-kvstore" {
         set(key: string, value: V, options?: {}): Promise<string>;
         
         del(key: string, options?: {}): Promise<string>;
+
+        all: {[key: string]: V};
     }
 }
```
Comment (tabcat) [inline KeyValueStore.d.ts:14]: new a newline character here

Comment (tabcat) [inline KeyValueStore.d.ts:14]: a newline character is missing at the end of the file

Comment (tabcat) [reply KeyValueStore.d.ts:14]: looks like this existed before you, check `files changed` to see it.

**Comment by tabcat** [comment]
@chrispanag do you have time to check this out?

**Comment by chrispanag** [comment]
Yeap! I'll try to check it out by tomorrow.

**Review by chrispanag — APPROVED** [review]
Seems good, this change was definitely needed.

**Comment by tabcat** [comment]
I don't have access to the repo so I'll have to get someone to merge it.

**Comment by tabcat** [comment]
This repo isnt published to npm for some reason so I'll make a pr to add publishing workflows.

**Comment by chrispanag** [comment]
> This repo isnt published to npm for some reason so I'll make a pr to add publishing workflows.

yeah, we are pulling it through github 😅 maybe add it to the [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) repo as `@types/orbit-db`? (makes more sense for the typescript ecosystem dwellers.

**Comment by tabcat** [comment]
it looks like @types/orbit-db is already the package name so no worries there.

**Pull request merged**